app/configuration/Config.scala

```diff
@@ -110,33 +112,39 @@ object Config {
       codes = PromoCodeSet(PromoCode("DGA88"), PromoCode("DGB88")),
       description = "Get £30 to spend with a top retailer of your choice when you subscribe. Use your digital gift card at John Lewis, Amazon, M&S and more. Treat yourself or a friend.",
       expires = new LocalDate(2016,4,1).toDateTime(LocalTime.Midnight, DateTimeZone.forID("Europe/London")),
-      imageUrl = "https://media.guim.co.uk/076bb31be49a31dfe82869ed2937fc8254917361/0_0_850_418/850.jpg",
-      promotionType = Incentive,
-      redemptionInstructions = "We'll send redemption instructions to your registered email address",
+      imageUrl = Some("https://media.guim.co.uk/076bb31be49a31dfe82869ed2937fc8254917361/0_0_850_418/850.jpg"),
+      promotionType = Incentive(
+        redemptionInstructions = "We'll send redemption instructions to your registered email address",
+        termsAndConditions = "<h4>Giftcloud £30 gift card terms and cond'itions</h4><p>Offer only available to customers who subscribe after trial period. Customers are asked to allow up to 35 days from first payment date to receive their gift card redemption email. Offer available to customers who subscribe after trial period only. Once customers have received their gift card redemption email they will have 90 days to claim the £30 gift card by selecting their chosen digital gift card and entering their email or phone number after which time the gift will no longer be available. In the event stock runs out you may be offered an alternative gift of a similar value or a full refund. GNM reserves the right to withdraw this promotion at any time.</p> <p>Amazon.co.uk Gift Cards (“GCs”) sold by Giftcloud, an authorised and independent reseller of Amazon.co.uk Gift Cards. Amazon.co.uk Gift Cards may be redeemed on the Amazon.co.uk website towards the purchase of eligible products listed in our online catalogue and sold byAmazon.co.uk or any other seller selling through Amazon.co.uk. GCs cannot be reloaded, resold, transferred for value, redeemed for cash or applied to any other account. Amazon.co.uk is not responsible if a GC is lost, stolen, destroyed or used without permission. See <a href='http://www.amazon.co.uk/gc-legal' target='_blank'>www.amazon.co.uk/gc-legal</a> for complete terms and conditions. GCs are issued by Amazon EU S.à r.l. All Amazon ®, ™ & © are IP of Amazon.com, Inc. or its affiliates.</p>"
+      ),
       roundelHtml = "Free <span class='roundel__strong'>£30</span> digital gift card",
       thumbnailUrl = "http://lorempixel.com/46/16/abstract",
       title = "Free £30 digital gift card when you subscribe"
     )
   }
 
-  def discountPromo(env: String): Option[Promotion] = {
+  def discountPromo(env: String): Option[AnyPromotion] = {
     val prpIds = digipackRatePlanIds(env)
+    val country = CountryGroup.UK
+    val discountedPrice = Price(9.99f, country.currency)
+    val originalPrice = Price(11.99f, country.currency)
```

Comment:
I still think we should hard code the output of this price-to-percentage calculation, lest we become tempted to use it more permanently